tests/test_inference.py

```diff
@@ -1053,14 +1038,14 @@ def test_from_historical_tree_sequence(self):
             assert tsutil.json_metadata_is_subset(i1.metadata, i2.metadata)
 
 
-class TestThreads(TsinferTestCase):
+class TestThreads:
     def test_equivalance(self):
         rho = 2
         ts = msprime.simulate(5, mutation_rate=2, recombination_rate=rho, random_seed=2)
         sample_data = tsinfer.SampleData.from_tree_sequence(ts)
         ts1 = tsinfer.infer(sample_data, num_threads=1)
         ts2 = tsinfer.infer(sample_data, num_threads=5)
-        self.assertTreeSequencesEqual(ts1, ts2)
+        ts1.equals(ts2, ignore_provenance=True)
```

Comment:
Missing an assert here, this test doesn't actually check anything now.

Reply:
Ha. Good spot. Sorry!